roles/openshift_metrics/templates/hawkular_cassandra_rc.j2

```diff
@@ -120,7 +120,11 @@ spec:
           exec:
             command:
             - "/opt/apache-cassandra/bin/cassandra-docker-ready.sh"
-          timeoutSeconds: 10
+          timeoutSeconds: "{{openshift_metrics_cassandra_readprobe_timeout}}"
```

Comment:
i would add a ` | default(10)` here just in case

Reply:
Done!